File: server/AppSupport.Data/Entities/ManifestPlane.cs
using System.Collections.Generic;

namespace AppSupport.Data.Entities
{
    public class ManifestPlane
    {
        public int Id { get; set; }
        public int ManifestId { get; set; }
        public int PlaneId { get; set; }

        public Manifest Manifest { get; set; }
        public Plane Plane { get; set; }

        public IEnumerable<ManifestPerson> ManifestPeople { get; set; }
    }
}

File: client/core/models/api/manifest-plane.ts
import { Manifest } from './manifest';
import { ManifestPerson } from './manifest-person';
import { Plane } from './plane';

export interface ManifestPlane {
  id: number;
  manifestId: number;
  planeId: number;

  manifest: Manifest;
  plane: Plane;

  manifestPeople: ManifestPerson[];
}
